bullet_train-obfuscates_id/app/models/concerns/obfuscates_id.rb

```diff
@@ -46,6 +50,7 @@ def obfuscated_id
   end
 
   def to_param
-    obfuscated_id
+    # Even if the `slug` column exists, it might be empty (i.e. - Team).
```

Comment:
We have a `slug` column on `Team` from a really long time ago:

https://github.com/bullet-train-co/bullet_train/blob/6c11323d5fbeb93eb6974e4d67b6531651d3b9c6/db/migrate/20161117154605_create_teams.rb#L5